Source/WebCore/bindings/js/JSSubscriberCustom.cpp

```diff
@@ -31,12 +31,6 @@ namespace WebCore {
 template<typename Visitor>
 void JSSubscriber::visitAdditionalChildren(Visitor& visitor)
 {
-    if (auto* next = wrapped().nextCallbackConcurrently())
-        next->visitJSFunction(visitor);
-    if (auto* error = wrapped().errorCallbackConcurrently())
-        error->visitJSFunction(visitor);
-    if (auto* complete = wrapped().completeCallbackConcurrently())
-        complete->visitJSFunction(visitor);
```

Comment:
`Subscriber` no longer holds onto these, instead `InternalObserverFromScript` does. Is there a way I should traverse and mark the objects held in that class?

Reply:
You would probably need to call something like `wrapped().observer().visitAdditionalChildren` and then you can define the visitation function on the `InternalObserver` object, keeping in mind thread safety.

Reply:
There's a few ways to do it so you'd want to consider which objects are keeping other objects alive and then use the owning object to manage the lifetime of the owned object.

Reply:
One concern I had with this is that it's not possible to build virtual template functions, so I was unsure how this could be done with each specialisation.

Reply:
You'd need to do what `ActiveDOMCallback` does and define them as overloads instead of templates.